src/box/box.cc

```diff
@@ -3545,7 +3583,7 @@ bootstrap(const struct tt_uuid *instance_uuid,
 	assert(master == NULL || master->applier != NULL);
 
 	if (master != NULL && !tt_uuid_is_equal(&master->uuid, &INSTANCE_UUID)) {
-		bootstrap_from_master(master);
+		rebootstrap_from_master(master);
```

Comment:
`box_restart_replication()` and `replicaset_find_join_master()` are called above, and they will be called again in `rebootsrap_from_master()`.
This means that the piece of code above should be part of `rebootstrap_from_master()`.
Same about the piece below. With `bootstrap_master()`

Reply:
Done